torch/csrc/jit/pickler.h

```diff
@@ -222,5 +250,12 @@ class Unpickler {
   OpCode last_opcode_;
 };
 
+// returns (record_size, data_ptr) for a tensor, converting it to a CPU tensor
+// if necessary
+std::pair<at::Tensor, uint64_t> getWriteableTensor(const at::Tensor& tensor);
+
+// return a unique ID for this tensor
```

Comment:
This comment is also misleading. Two tensors that are not the same can have the same key.  This should be getStorageKey with a comment explaining what it does.